.cg-deploy/manifests/staging/manifest-frontend-staging.yml

```diff
@@ -1,9 +1,9 @@
 ---
 applications:
-- name: forest-service-trees-staging
+- name: open-forest-platform-frontend-staging
   path: ../../../frontend/dist
   memory: 64M
-  host: forest-service-trees-staging
+  host: open-forest-dev
```

Comment:
should the host reflect `staging` instead of `dev`?

Reply:
ah, yes, this is what @ASprinkle preferred too.